[Merge-Queue] Canonicalize commit without rebase

JonWBedard opened the pull request:
#### 982cb855332505a205085636b8fbab4f989d2a86
<pre>
[Merge-Queue] Canonicalize commit without rebase
<a href="https://bugs.webkit.org/show_bug.cgi?id=238614">https://bugs.webkit.org/show_bug.cgi?id=238614</a>
&lt;rdar://problem/91113465 &gt;

Reviewed by Aakash Jain.

This is helpful immiediatly after landing a subversion commit
and allows us to bypass the 60 second sleep in PushCommitToWebKitRepo.

* Tools/CISupport/ews-build/steps.py:
(Canonicalize.__init__): Allow caller to opt out of rebasing.
(Canonicalize.run): Ditto.
* Tools/CISupport/ews-build/steps_unittest.py:

Canonical link: <a href="https://commits.webkit.org/249136@main">https://commits.webkit.org/249136@main</a>



Canonical link: <a href="https://commits.webkit.org/249136@main">https://commits.webkit.org/249136@main</a>
git-svn-id: <a href="https://svn.webkit.org/repository/webkit/trunk@292235">https://svn.webkit.org/repository/webkit/trunk@292235</a> 268f45cc-cd09-0410-ab3c-d52691b4dbfc
</pre>

aj062 commented on Tools/CISupport/ews-build/steps.py:
```diff
@@ -4818,20 +4818,26 @@ class Canonicalize(steps.ShellSequence, ShellMixin):
     haltOnFailure = True
 
-    def __init__(self, **kwargs):
+    def __init__(self, rebase=True, **kwargs):
```
Nit: maybe rebase => rebase_enabled

JonWBedard committed:
[Merge-Queue] Canonicalize commit without rebase
https://bugs.webkit.org/show_bug.cgi?id=238614
<rdar://problem/91113465>

Patch by Jonathan Bedard <jbedard@apple.com> on 2022-04-01
Reviewed by Aakash Jain.

This is helpful immiediatly after landing a subversion commit
and allows us to bypass the 60 second sleep in PushCommitToWebKitRepo.

* Tools/CISupport/ews-build/steps.py:
(Canonicalize.__init__): Allow caller to opt out of rebasing.
(Canonicalize.run): Ditto.
* Tools/CISupport/ews-build/steps_unittest.py:
